.gitignore

```diff
@@ -111,4 +111,5 @@ build-artifacts.tar.gz
 
 # sample app resources
 sample*
-.graphqlconfig.yml
\ No newline at end of file
+.graphqlconfig.yml
+aws-datastore/src/androidTest/res/raw/amplifyconfigurationupdated.json
```

Comment:
May not be a bad idea to move to `**/awsconfiguration.json` in the section where we exclude these similar files. Also not sure what is generating this? Not holding up approval on this though.

Reply:
I added that because that file kept getting adding accidentally.